Packs/CommonTypes/IncidentFields/incidentfield-cloudregionlist.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_cloudregionlist",
+    "version": -1,
+    "modified": "2022-09-28T13:34:19.571020931Z",
+    "name": "Cloud Region List",
+    "ownerOnly": false,
+    "cliName": "cloudregionlist",
+    "type": "multiSelect",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "openEnded": true,
+    "associatedTypes": [
+        "Cortex XDR - XCLOUD"
```

Comment:
need to add the new type you created as well

Reply:
fixed